interpreter/cling/include/cling/Interpreter/RuntimeUniverse.h

```diff
@@ -176,7 +176,7 @@ extern "C" {
   /// run cling in -no-rtti mode.
   ///
 
-  void cling_runtime_internal_throwIfInvalidPointer(void* Sema,
+  void* cling_runtime_internal_throwIfInvalidPointer(void* Sema,
```

Comment:
The decl returns `void*`, the impl `const void*`. Both should probably return `void*`.